user/views.py

```diff
@@ -116,4 +121,94 @@ def get(self, request):
 
         except KeyError:
             return JsonResponse({'message':'KEY_ERROR'}, status=404)
-        
+
+class ProfileUpload(View):
+    __client = boto3.client(
+            's3',
+            aws_access_key_id     = MY_AWS_ACCESS_KEY_ID,
+            aws_secret_access_key = MY_AWS_SECRET_ACCESS_KEY  
+            )
```

Comment:
거의 다 왔습니다!! boto3 client 변수 선언하는 부분만 class 밖으로 꺼내면 더 좋을 것 같아요!
`ProfileUpload`과 `ProfileUploadUpdate`에서 같이 사용할 수 있도록이요!
```suggestion
client = boto3.client(...)
class Profile ..

class ...
```